urls.py

```diff
@@ -21,6 +21,6 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^cfp($|/)', ProposalCreate.as_view(success_url='/'), name="cfp"),
```

Comment:
oh, you mean the regex is not correct? I agree we should end it somewhere. Is it okay to be consistent and just not require an ending slash?